m4/libcurl.m4

```diff
@@ -62,7 +62,7 @@ AC_DEFUN([LIBCURL_CHECK_CONFIG],
   AH_TEMPLATE([LIBCURL_FEATURE_KRB4],[Defined if libcurl supports KRB4])
   AH_TEMPLATE([LIBCURL_FEATURE_IPV6],[Defined if libcurl supports IPv6])
   AH_TEMPLATE([LIBCURL_FEATURE_LIBZ],[Defined if libcurl supports libz])
-  AH_TEMPLATE([LIBCURL_FEATURE_ASYNCHDNS],[Defined if libcurl supports AsynchDNS])
+  AH_TEMPLATE([LIBCURL_FEATURE_ASYNCHDNS],[Defined if libcurl supports AsyncDNS])
```

Comment:
I probably should have left this file alone.

https://github.com/curl/curl/blob/50a1c30b91f91139a13ec352d08c383699b183df/docs/libcurl/libcurl.m4#L66

I'll drop it. (I have sent curl PRs...)